packages/core/src/app.ts

```diff
@@ -944,7 +944,9 @@ let d = w.define;
 {{/if}}
 
 {{#if autoRun ~}}
-  i("{{js-string-escape mainModule}}").default.create({{{json-stringify appConfig}}});
+  if (!window.disableAutoBoot) {
```

Comment:
this will also need a test

Reply:
I didn't see any tests for the template generated here. Because of that didn't add a test. 

As I am not familiar adding tests for `broccoli-plugin` build, can you refer me to an example that I can use. 

I tested in the sample ember App by linking to the changes. And didn't see any issues.

Reply:
@stefanpenner I am not sure if there is an easy way of testing this global flag setup without adding a sample package with fastboot related changes. But Fastboot updates depends on this PR related to config-module - https://github.com/embroider-build/embroider/pull/224

if above PR looks good, Can we have above PR merged?